### ISSUE TITLE
Statefulset feature not working

### ISSUE THREAD

**Issue by a1ii54** [harbur/kubernetic#59]
Hi

It seems the new statefulset feature that was added in 2.0.15 is not working.
I can see my statefulsets in the web ui but not in the statefulset section of kubernetic.

Is there perhaps a version compatibility that I should be aware of?

Kind regards
Jarred Naidoo

**Comment by spiddy** [comment]
Do you have privileges locally to see statefulsets?

Try the following on CLI, the web ui (I assume you refer to dashboard) may use different authentication:

```
kubectl get statefulsets
```

**Comment by a1ii54** [comment]
Hi spiddy, thanks for the response.

That command does work for me

![image](https://user-images.githubusercontent.com/25863579/43944465-4b8fc572-9c7f-11e8-896e-3bfbf4204d58.png)

**Comment by spiddy** [comment]
Kubernetic can now create StatefulSets from the UI. Can you try to create a StatefulSet and see if the generated StatefulSet can be viewed on the UI?

**Comment by a1ii54** [comment]
No luck....
![image](https://user-images.githubusercontent.com/25863579/47288436-6819ea00-d5f6-11e8-926e-60fe74bbe805.png)

I'm running kubernetic 2.2.0 on windows.
Kubernetes v1.8.7

Is there extra logging or error reports for kubernetic somewhere?

**Comment by spiddy** [comment]
Yes you can check in this directory: `~/Library/Logs/Kubernetic/`

**Comment by spiddy** [comment]
The StatefulSet API version used on Kubernetic is `apps/v1`, can you check the version of StatefulSet resources in your cluster it's probably previous/beta one.

**Comment by spiddy** [comment]
As stated in https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/
> Note: StatefulSets are stable (GA) in 1.9.

The current solution depends on specific API version.

This can be improved by two ways:
- Kubernetic should either notify user that the API is not sufficiently new.
- Or it should not depend on specific API version (better one)

let's keep this issue open until one of the above is put in place
